third_party/nvidia/backend/compiler.py

```diff
@@ -265,7 +265,8 @@ def make_llir(src, metadata, options, capability):
         features = get_features(options)
         triple = 'nvptx64-nvidia-cuda'
         llvm.attach_datalayout(llvm_mod, triple, proc, features)
-        nvidia.set_nvvm_reflect_ftz(llvm_mod)
+        if os.environ.get("TRITON_FTZ", "1") != "0":
```

Comment:
you'll want to add this env var to the list of var that gets hashed in the cache key otherwise this is going to cause caching problems.
Env var are usually for debugging purposes, is this mostly for experimenting?

Reply:
Added to CACHE_INVALIDATING_ENV_VARS!

It's an important thing to have control over for real kernels, but we'll (probably?) never need to run at scale with this option off. So up to you if this is good enough.

Reply:
sounds good to me. If I can add one more nit maybe we could pick a more descriptive name? maybe TRITON_DENORM_FTZ?

Reply:
sure, that sounds good

btw, the nvcc option is just `-ftz` so i don't think it'll be super confusing